tpm2/src/getkey/main.c

```diff
@@ -229,11 +275,18 @@ int main(int argc, char **argv)
     BIO_free_all(bio);
     SSL_CTX_free(ctx);
     kmyth_clear_and_free(clientPrivateKey_data, clientPrivateKey_size);
+    free(sdo_orig_fn);
+    if (authString != NULL)
```

Comment:
After the call to tpm2_kmyth_unseal(), we should no longer need the authString or ownerAuthPasswd variables. If we moved this to immediately follow the if statement for that call (lines 246-261), we could go ahead and clear them earlier, eliminating the need for a couple of checks/calls to clear them later in the code.